backend.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/crowdsecurity/crowdsec/pkg/sqlite"
+	log "github.com/sirupsen/logrus"
+)
+
+type backend interface {
+	Init() error
+	ShutDown() error
+	Run(*sqlite.Context, time.Duration) error
+}
+
+type backendCtx struct {
+	ctx backend
+}
+
+func (b *backendCtx) Init() error {
+	err := b.ctx.Init()
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (b *backendCtx) Run(dbCTX *sqlite.Context, frequency time.Duration) error {
+	err := b.ctx.Run(dbCTX, frequency)
+	if err != nil {
+		log.Fatalf(err.Error())
```

Comment:
maybe return error instead of fatal'ing here

Reply:
i don't think we want to return an error here since this function is called in a go routine